### DISCUSSION TITLE
How to tell it to install to `/usr/local`?

### DISCUSSION THREAD

**Discussion by ghost** [lcn2/calc#105] [Q&A] [answered]
The command `make clobber all T=/usr/local` doesn't work, see: https://github.com/lcn2/calc/issues/104#issuecomment-1746129771

It seems I have to edit `Makefile.local`?

**Comment by lcn2** [comment]
The `T` Makefile variable is more for doing cross compiler builds, and building things like RPM where instead of installing on the primary system, you might with to install under a development tree such as:

```sh
make clobber all chk T=/var/tmp/calc
sudo make install T=/var/tmp/calc
```

Since `PREFIX` defaults to `/usr/local`, setting `T` to `/var/tmp/calc` would result in calc being installed under `/var/tmp/calc/usr/local`.  So think of `T` as your "root" and `PREFIX` as the main paths under your "root".

**Comment by ghost** [comment]
@lcn2 For unknown reasons, it will always install to `/usr` regardless of whether `PREFIX` is set or not. Please have a look at it. Thank you.

**Comment by lcn2** [comment, marked as answer]
Our previous answer was incorrect, sorry.

Please see issue #108 and in particular [comment 1748621729](https://github.com/lcn2/calc/issues/108#issuecomment-1748621729), and [comment 748629904](https://github.com/lcn2/calc/issues/108#issuecomment-1748629904), and [comment 1748641721](https://github.com/lcn2/calc/issues/108#issuecomment-1748641721).

We have added a TODO about how and where calc installs things in order to improve the process and documentation.  See [comment 1925271244)](https://github.com/lcn2/calc/issues/103#issue-1925271244).

Sorry @iahung2, for the confusion: we forgot about the 4 parameters BINDIR, LIBDIR, CALC_SHAREDIR, and CALC_INCDIR and that non-MacOS system set those to install under /usr.  In the above comments we showed how to change them on both the command line and via `Makefile.local`.
